src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationEventService.kt

```diff
@@ -126,7 +128,11 @@ class VisitNotificationEventService(
   }
 
   fun handlePrisonerReceivedNotification(notificationDto: PrisonerReceivedNotificationDto) {
-    // TODO not yet implemented
+    LOG.debug("PrisonerReceivedNotification notification received : {}", notificationDto)
+    if (PrisonerReceivedReasonType.TRANSFERRED == notificationDto.reason) {
```

Comment:
should we check if its an actively supported prison as well? Probably missing in all current handleEvent methods.

Reply:
This will be implemented in VB-3939. Thanks Dhiraj